.env.example

```diff
@@ -1,8 +1,13 @@
 # REQUIRED if you're self-hosting
 RAILS_ENV= # set to 'production' or 'development'
+RACK_ENV=
+DATABASE_HOST=
 DATABASE_NAME=
 DATABASE_USER=
 DATABASE_PASSWORD=
+POSTGRES_PASSWORD=
+POSTGRES_USER=
+POSTGRES_DB=
 REDIS_PORT=
```

Comment:
We can remove `REDIS_PORT` in this case, since it's already included in `REDIS_URL` now.